bin/pt-kill

```diff
@@ -4,6 +4,7 @@
 # See "COPYRIGHT, LICENSE, AND WARRANTY" at the end of this file for legal
 # notices and disclaimers.
 
+use DBI;
```

Comment:
This is auto-generated code. No change should be here.

I also do not understand why you need to explicitly import DBI for the code you propose.